tests/garage/torch/misc/test_tensor_utils.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pytest
+import tensorflow as tf
+import torch
+
+import garage.tf.misc.tensor_utils as tf_utils
+import garage.torch.misc.tensor_utils as torch_utils
+from tests.fixtures import TfGraphTestCase
+
+
+class TestTensorUtil(TfGraphTestCase):
+    # yapf: disable
+    @pytest.mark.parametrize('rewards_val, baselines_val', [
+        (np.ones((4, 6)), np.zeros((4, 6))),
+        (np.random.randn(4, 6), np.random.randn(4, 6)),
```

Comment:
please don't use randomness in tests. if you need to, you can instead generate a static random data set, or use alternatives like digits of pi or fibonacci sequences.

Reply:
done